ballerina/Dependencies.toml

```diff
@@ -28,7 +28,7 @@ modules = [
 [[package]]
 org = "ballerina"
 name = "io"
-version = "1.3.1"
+version = "1.4.0"
```

Comment:
Shall we change the distribution in here as well?
https://github.com/ballerina-platform/module-ballerina-file/blob/5c002f3b73d4cb972fa52b2d727d80083f4692cf/build-config/resources/Ballerina.toml#L10

Reply:
Updated in [ba252f9](https://github.com/ballerina-platform/module-ballerina-file/pull/475/commits/ba252f947b23fd76d69c8220f08fc165d4d62924)